pydis_site/apps/api/tests/test_users.py

```diff
@@ -431,18 +431,21 @@ def test_no_metricity_user(self):
 
         # When
         url = reverse('bot:user-metricity-data', args=[0], host='api')
-        response = self.client.get(url)
+        response1 = self.client.get(url)
+        url = reverse('bot:user-metricity-review-data', args=[0], host='api')
+        response2 = self.client.get(url)
 
         # Then
-        self.assertEqual(response.status_code, 404)
+        self.assertEqual(response1.status_code, 404)
+        self.assertEqual(response2.status_code, 404)
```

Comment:
It'd be better to make a separate test for each endpoint. In any case, the names `response1` and `response2` aren't great.